tools/functional-tester/util/checks.go

```diff
@@ -12,26 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package util
```

Comment:
make this `package check`? `Checker` doesn't seem like it belongs in a packaged called `util`